src/Mailchimp/Campaign/Listener/SetCampaignReplyToListener.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace AppBundle\Mailchimp\Campaign\Listener;
+
+use AppBundle\AdherentMessage\AdherentMessageTypeEnum;
+use AppBundle\Mailchimp\Events;
+use AppBundle\Mailchimp\RequestEvent;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+class SetCampaignReplyToListener implements EventSubscriberInterface
```

Comment:
I know in the project, a lot of subscribers are suffixed by `Listener`.

What do you think to prefix now all new event subscribers  by `Subscriber` ?

Reply:
why not, done